link/uprobe.go

```diff
@@ -159,6 +159,48 @@ func (ex *Executable) load(f *internal.SafeELFFile) error {
 	return nil
 }
 
+// address calculates the address of a symbol in the executable.
+//
+// opts must not be nil.
+func (ex *Executable) addressMulti(symbols []string) ([]uint64, error) {
```

Comment:
Why can't this call `ex.address` in a loop instead of copying the body?

Reply:
yes, we can do that, will change